collector/src/bin/rustc-perf-collector/main.rs

```diff
@@ -240,7 +244,7 @@ fn main_result() -> Result<i32, Error> {
     )?;
     let use_remote = matches.is_present("sync_git");
     let out_repo = PathBuf::from(matches.value_of_os("output_repo").unwrap());
-    let mut out_repo = outrepo::Repo::open(out_repo, use_remote)?;
+    let mut out_repo = outrepo::Repo::open(out_repo, true, use_remote)?;
```

Comment:
Could we condition the `true` being passed here to be only if we're running locally? I'd like to avoid accidentally clearing out some git directory on the perf collector...

Reply:
Sure.